examples/tests/cypress/integration/channels.spec.js

```diff
@@ -0,0 +1,31 @@
+describe('Channels', () => {
+  ['sse', 'ws'].forEach(type => {
+    it(`${type} Public events`, () => {
```

Comment:
"it sse public events" is not the most descriptive test name. What does this test? Does `connectAndSendEvents` also check that the message sent has been received afterwards? If yes, shouldn't the function name hint on that? If not, then this test only tests that sending something "works".. probably.. but I don't see any assertions ;)

Reply:
ye, I guess it's poor naming, I'll change it -- you can find assertions in `commands.js`

Reply:
Updated structure and naming in general, should be easier to follow now, let me know if it's enough or you see some places for improvement.

Reply:
👍

Reply:
very cool - way more readable now!